apps/Gruntfile.js

```diff
@@ -345,12 +345,18 @@ module.exports = function (grunt) {
           ' -p [ factor-bundle -o ' + allFilesDest.join(' -o ') + ' ] -o ' + outputDir + 'common.js' :
           ' -o ' + allFilesDest[0]
       );
+  var applabAPIExec = 'mkdir -p build/browserified && `npm bin`/browserifyinc' +
+      ' --cachefile ' + outputDir + 'applab-api-cache.json' +
+      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+      (envOptions.dev ? '' : ' -t [ envify --NODE_ENV production ]') +
```

Comment:
I probably changed this out from under you today - please update this transform to use `loose-envify` instead of `envify` and instead of passing NODE_ENV=production as configuration to the transform set it on the environment right in front of the build command.  See how browserifyExec does it now as an example.